storio-common/src/main/java/com/pushtorefresh/storio/operations/PreparedOperation.java

```diff
@@ -62,4 +62,7 @@
     @NonNull
     @CheckResult
     Single<Result> asRxSingle();
+
+    @NonNull
+    Object getData();
```

Comment:
I think we need to try generics in prepared operations, `Object` seems too weird…